src/renderer/components/data-settings/data-settings.js

```diff
@@ -882,87 +882,116 @@ export default defineComponent({
 
       const requiredKeys = [
         'playlistName',
-        'videos'
+        'videos',
       ]
 
       const optionalKeys = [
+        'description',
+        'createdAt',
+      ]
+
+      const ignoredKeys = [
         '_id',
+        'title',
+        'type',
         'protected',
-        'removeOnWatched'
+        'lastUpdatedAt',
+        'lastPlayedAt',
+        'removeOnWatched',
+
+        'thumbnail',
+        'channelName',
+        'channelId',
+        'playlistId',
+        'videoCount',
       ]
 
       const requiredVideoKeys = [
         'videoId',
         'title',
         'author',
         'authorId',
-        'published',
         'lengthSeconds',
         'timeAdded',
-        'isLive',
-        'type',
+
+        // `playlistItemId` should be optional for backward compatibility
```

Comment:
Shouldn't this be added to optionalKeys then? Or is there a reason it wasn't added?

Reply:
If you look at the video import code, the whole video object is pushed as long as all the required keys are present (not whitelisting properties)